### PR TITLE
[sonic-py-common] Relocate some functions from sonic-utilities

### PR DESCRIPTION
**Pull request by judyjoseph**
Move the common multi-asic/general functions from sonic-utilities to py-common 
validate_namespace()
get_port_table_name()
get_interface_table_name()

### CODE REVIEW

File: src/sonic-py-common/tests/interface_test.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+# TODO: Remove this if/else block once we no longer support Python 2
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    # Expect the 'mock' package for python 2
+    # https://pypi.python.org/pypi/mock
+    import mock
```
Comment (jleveque) [inline src/sonic-py-common/tests/interface_test.py:10]: It appears this is not being used in this file. Suggest to remove. We can add it back if we need to do mocking in this file at a later time.

File: src/sonic-py-common/tests/interface_test.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+from sonic_py_common import interface
+
+
+# TODO: Remove this if/else block once we no longer support Python 2
+if sys.version_info.major == 3:
+    BUILTINS = "builtins"
+else:
+    BUILTINS = "__builtin__"
```
Comment (jleveque) [inline src/sonic-py-common/tests/interface_test.py:19]: It appears this is not being used in this file. Suggest to remove. We can add it back if we need to do mocking in this file at a later time.

Comment (judyjoseph) [reply src/sonic-py-common/tests/interface_test.py]: Sure, removed unused imports from the interface_test.

**Comment by judyjoseph** [comment]
retest this please

**Pull request merged**